addons/l10n_de/migrations/11.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,146 @@
+NEW account.account.tag: l10n_de.tag_de_pl_02
+NEW account.account.tag: l10n_de.tag_de_pl_03
+NEW account.account.tag: l10n_de.tag_de_pl_04
+NEW account.account.tag: l10n_de.tag_de_pl_05
+NEW account.account.tag: l10n_de.tag_de_pl_06
+NEW account.account.tag: l10n_de.tag_de_pl_07
+NEW account.account.tag: l10n_de.tag_de_pl_08_1
+NEW account.account.tag: l10n_de.tag_de_pl_08_2
+NEW account.account.tag: l10n_de.tag_de_pl_08_3
+NEW account.account.tag: l10n_de.tag_de_pl_08_4
+NEW account.account.tag: l10n_de.tag_de_pl_08_5
+NEW account.account.tag: l10n_de.tag_de_pl_08_6
+NEW account.account.tag: l10n_de.tag_de_pl_08_7
+NEW account.account.tag: l10n_de.tag_de_pl_09
+NEW account.account.tag: l10n_de.tag_de_pl_10
+NEW account.account.tag: l10n_de.tag_de_pl_11
+NEW account.account.tag: l10n_de.tag_de_pl_12
+NEW account.account.tag: l10n_de.tag_de_pl_13
+NEW account.account.tag: l10n_de.tag_de_pl_14
+NEW account.account.tag: l10n_de.tag_de_pl_15
+DEL account.account.tag: l10n_de.tag_de_01
+DEL account.account.tag: l10n_de.tag_de_02
+DEL account.account.tag: l10n_de.tag_de_03
+DEL account.account.tag: l10n_de.tag_de_04
+DEL account.account.tag: l10n_de.tag_de_05
+DEL account.account.tag: l10n_de.tag_de_06
+DEL account.account.tag: l10n_de.tag_de_07
+DEL account.account.tag: l10n_de.tag_de_08
+DEL account.account.tag: l10n_de.tag_de_09
+DEL account.account.tag: l10n_de.tag_de_10
+DEL account.account.tag: l10n_de.tag_de_11
+DEL account.account.tag: l10n_de.tag_de_12
+DEL account.account.tag: l10n_de.tag_de_13
+DEL account.account.tag: l10n_de.tag_de_14
+DEL account.account.tag: l10n_de.tag_de_15
+DEL account.account.tag: l10n_de.tag_de_16
+DEL account.account.tag: l10n_de.tag_de_17
+DEL account.account.tag: l10n_de.tag_de_18
+DEL account.account.tag: l10n_de.tag_de_19
+DEL account.account.tag: l10n_de.tag_de_20
+DEL account.account.tag: l10n_de.tag_de_22
+DEL account.account.tag: l10n_de.tag_de_23
+DEL account.account.tag: l10n_de.tag_de_24
+DEL account.account.tag: l10n_de.tag_de_25
+DEL account.account.tag: l10n_de.tag_de_26
+DEL account.account.tag: l10n_de.tag_de_27
+DEL account.account.tag: l10n_de.tag_de_28
+DEL account.account.tag: l10n_de.tag_de_29
+DEL account.account.tag: l10n_de.tag_de_30
+DEL account.account.tag: l10n_de.tag_de_31
+DEL account.account.tag: l10n_de.tag_de_32
+DEL account.account.tag: l10n_de.tag_de_33
+DEL account.account.tag: l10n_de.tag_de_34
+DEL account.account.tag: l10n_de.tag_de_37
+DEL account.account.tag: l10n_de.tag_de_38
+DEL account.account.tag: l10n_de.tag_de_40
+DEL account.account.tag: l10n_de.tag_de_50
+DEL account.account.tag: l10n_de.tag_de_51
+DEL account.account.tag: l10n_de.tag_de_54
+DEL account.account.tag: l10n_de.tag_de_55
+DEL account.account.tag: l10n_de.tag_de_56
+DEL account.account.tag: l10n_de.tag_de_57
+DEL account.account.tag: l10n_de.tag_de_58
+DEL account.account.tag: l10n_de.tag_de_70
+DEL account.account.tag: l10n_de.tag_de_71
+DEL account.account.type: l10n_de.account_type_other (noupdate)
```

Comment:
It seems that this old type has been replaced by `account.data_account_type_other_income` (I have checked several accounts, but not all). I think you should search all occurrences of your account.account with this `user_type_id` and replace them with the other type, and finally remove the account type.

Reply:
I'm not 100% sure here. I also went through the account types and this one is the only one I can imagine to be right here. I found the PR https://github.com/odoo/odoo/pull/17447 and the discussion in https://github.com/odoo/odoo/issues/18571 but couldn't find out why it was removed.

Nevertheless I wrote a script to replace the type with the other one.